src/nvim/getchar.c

```diff
@@ -3030,31 +3033,6 @@ int buf_do_map_explicit(int maptype, MapArguments *args, int mode,
   return retval;
 }
 
-/// Like @ref do_map, but you can specify the target buffer.
-int buf_do_map(int maptype, char_u *arg, int mode, bool is_abbrev, buf_T *buf)
```

Comment:
- This was only used in `do_map` so I moved it there. 
- Renamed `buf_do_map_explicit` to `buf_do_map`